core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -235,10 +233,10 @@ public WorkerNetAddress getFieldValue(MasterWorkerInfo o) {
   @Override
   public Map<String, TProcessor> getServices() {
     Map<String, TProcessor> services = new HashMap<>();
-    services.put(Constants.BLOCK_MASTER_CLIENT_SERVICE_NAME,
-        new BlockMasterClientService.Processor<>(new BlockMasterClientServiceHandler(this)));
-    services.put(Constants.BLOCK_MASTER_WORKER_SERVICE_NAME,
-        new BlockMasterWorkerService.Processor<>(new BlockMasterWorkerServiceHandler(this)));
+    //services.put(Constants.BLOCK_MASTER_CLIENT_SERVICE_NAME,
```

Comment:
add TODO

Reply:
Being addressed as part of boot-up and security work.